Readme.md

```diff
@@ -74,34 +124,53 @@ Cloning...Succeeded
 Fetching commits and trees from origin (no cache server)...Succeeded
 Configuring Watchman...Succeeded.
 Validating repo...Succeeded
+```
+
+Then, navigate into the repository's `src` directory.
 
+```sh
 $ cd ForTests/src
 $ ls
 AuthoringTests.md  GvFlt_EULA.md  GVFS.sln  License.md  nuget.config  Protocol.md  Readme.md  Settings.StyleCop
+```
+
+This directory is the _working directory_, which contains all files
+tracked by Git. It also includes the `.git` directory and its contents.
+This allows build outputs to be written to directories next to the `src`
+directory instead of inside it, which speeds up some Git operations.
```

Comment:
> This allows build outputs to be written to directories next to the `src`
> directory instead of inside it, which speeds up some Git operations.

Where they put their build output and compiler trash is outside of our scope.
I mean, we're not going to write anything there or alter their build scripts to do so.
This sentence kinda implies that we might try something like that as part of the
magic that we're doing.  I'd suggest a wording that says that that directory is
available for build output if they want to use it -- but that makes their build scripts
complicated if some users are using Scalar and some are not.  Perhaps instead have
a brief sidebar on why you don't want the compiler/build system writing trash to the
source tree in general -- it's not necessarily particular to Scalar.